requirements.txt

```diff
@@ -0,0 +1 @@
+requests==2.18.4
```

Comment:
Maybe add a comment above this line to explain what this file is for?

Reply:
OK! changed it!